storage/storage_setup.py

```diff
@@ -0,0 +1,56 @@
+import sys
+from cassandra.cluster import Cluster
+from cassandra.auth import PlainTextAuthProvider
+
+#User arguments should contain the following content in that order: ip address, username, password, keyspace
+
+def cluster_setup(ip_address, username, password, keyspace):
+	auth_provider = PlainTextAuthProvider(username=username, password=password)
+	contact_point = []
+	contact_point.append(ip_address)
+	cluster = Cluster(contact_points=contact_point, auth_provider=auth_provider)
+	session = cluster.connect(keyspace)
+	return session
+
+
+def schema_setup(ip_addresses, username, password, keyspace):
+	session = cluster_setup(ip_addresses, username, password, keyspace)
+
+	session.execute(
+		"""CREATE TABLE IF NOT EXISTS relationship_primary_object_table(
+		object_id TEXT, 
+		relationship_type TEXT, 
+		relationship_value TEXT, 
+		timestamp timeuuid, 
+		weight int, 
+		PRIMARY KEY (object_id, relationship_type, relationship_value, timestamp)
```

Comment:
They end up as clustering columns and that should be helpful in querying. Also, I need timestamp to avoid potential duplicates in the PK.